src/components/order-service/order-service.controller.ts

```diff
@@ -185,7 +185,8 @@ export class OrderServiceController implements angular.IController {
       this.DataService
         .create('projectrequests', null, projReqObj, this.$scope)
         .then( (data: any) => {
-          this.ctrl.projectDisplayName = newProjDisplayName || newProjName;
+          this.ctrl.selectedProject = data;
+          this.ctrl.projectDisplayName = this.$filter('displayName')(this.ctrl.selectedProject) || this.ctrl.selectedProject.metadata.name;
```

Comment:
I think the `displayName` filter falls back to `metadata.name`, so the `||` shouldn't be necessary.